documentation/content/en/articles/committers-guide/_index.adoc

```diff
@@ -2020,6 +2022,95 @@ git log refs/freebsd/vendor/zlib/1.2.10
 
 would look at the log for the vendor branch for zlib starting at 1.2.10.
 
+=== Collaborating with others
+
+One of the keys to good software development on a project as large as FreeBSD is the ability to collaborate with others before you push your changes to the tree.
+The FreeBSD project's git repositories do not, yet, allow user created branches to be pushed to the repository, and therefore if you wish to share your changes with others you must use another mechanism, such as a hosted GitLab or GitHub, in order to share changes in a user generated branch.
+
+The following instructions show how to set up a user generated branch, based on the FreeBSD main branch, and push it to GitHub.
+
+Before you begin, make sure that your local git repo is up to date and has the correct origins set <<keeping_current,as shown above.>>
+
+[source,shell]
+````
+% git remote -v
+freebsd https://git.freebsd.org/src.git (fetch)
+freebsd ssh://git@gitrepo.freebsd.org/src.git (push)
+````
+
+The first step is to create a fork of https://github.com/freebsd/freebsd-src[FreeBSD] on GitHub following these https://docs.github.com/en/github/getting-started-with-github/fork-a-repo[guidelines].
+The destination of the fork should be your own, personal, GitHub account (gvnn3 in my case).
+
+Now add a remote on your local system that points to your fork:
+[source,shell]
+....
+% git remote add github git@github.com:gvnn3/freebsd-src.git
+% git remote -v
+github	git@github.com:gvnn3/freebsd-src.git (fetch)
+github	git@github.com:gvnn3/freebsd-src.git (push)
+freebsd	https://git.freebsd.org/src.git (fetch)
+freebsd	ssh://git@gitrepo.freebsd.org/src.git (push)
+....
+With this in place you can create a <<keeping_a_local_branch,as shown above.>>
+
+[source,shell]
+....
+% git checkout -b gnn-github
```

Comment:
I feel like this branch name might be confusing - maybe something like `gnn-fix` or `gnn-new-feature`?